package.xml

```diff
@@ -0,0 +1,93 @@
+    <contents>
+        <dir name="/">
+            <dir name="src">
+                <dir name="ext">
+                    <file name="compat_zend_string.c" role="src" />
+                    <file name="compat_zend_string.h" role="src" />
+                    <file name="ddtrace.c" role="src" />
+                    <file name="ddtrace.h" role="src" />
+                    <file name="debug.h" role="src" />
+                    <file name="dispatch.c" role="src" />
+                    <file name="dispatch.h" role="src" />
+                    <file name="dispatch_compat.h" role="src" />
+                    <file name="dispatch_compat_php5.c" role="src" />
+                    <file name="dispatch_compat_php5.h" role="src" />
+                    <file name="dispatch_compat_php7.c" role="src" />
+                    <file name="dispatch_setup.c" role="src" />
+                    <file name="version.h" role="src" />
+                </dir>
+            </dir>
+            <dir name="tests">
+                <dir name="ext">
+                    <file name="access_modifier_method_access_hook.phpt" role="test" />
+                    <file name="access_modifier_property_access_hook.phpt" role="test" />
+                    <file name="case_insensitive_method_hook.phpt" role="test" />
+                    <file name="closure_accessing_outside_variables.phpt" role="test" />
+                    <file name="closure_set_inside_object_methods.phpt" role="test" />
+                    <file name="method_returning_array.phpt" role="test" />
+                    <file name="multiple_instrumentations.phpt" role="test" />
+                    <file name="namespaces.phpt" role="test" />
+                    <file name="overriding_construct.phpt" role="test" />
+                    <file name="overriding_method_defined_in_parent.phpt" role="test" />
+                    <file name="private_method_hook.phpt" role="test" />
+                    <file name="protected_method_hook.phpt" role="test" />
+                    <file name="recursion.phpt" role="test" />
+                    <file name="return_value_passed.phpt" role="test" />
+                    <file name="simple_function_hook.phpt" role="test" />
+                    <file name="simple_method_hook.phpt" role="test" />
+                    <file name="throw_exception.phpt" role="test" />
+                    <file name="variable_length_parameter_list.phpt" role="test" />
+                    <file name="with_params_function_hook.phpt" role="test" />
+                    <file name="with_params_method_hook.phpt" role="test" />
+                </dir>
+            </dir>
+            <file name="CHANGELOG.md" role="doc" />
+            <file name="config.m4" role="src" />
+            <file name="LICENSE" role="doc" />
+            <file name="README.md" role="doc" />
+        </dir>
+    </contents>
+    <dependencies>
+        <required>
+            <php>
+                <min>5.6</min>
```

Comment:
can also set `<max>7.2.99</min>` as you known 7.3 is not supported

Reply:
Thanks! I just updated that until we can get PHP 7.3 support rocking. :)